python/lsst/ts/LinearStage/hardware.py

```diff
@@ -103,14 +98,30 @@ def __init__(self, port: str, address: int) -> None:
         }
         self.logger.debug("created LinearStageComponent")
 
-    def move_absolute(self, value):
-        """This method moves the linear stage absolutely by the number of steps away from the starting
-        position. i.e. value=10 would mean the stage would move 10 millimeters away from the start.
+    def configure(self, config):
+        self.port = config.port
+        self.address = config.address
+
+    def connect(self):
+        self.commander = zaber.AsciiDevice(self.port, self.address)
+
+    def disconnect(self):
+        self.commander.close()
+        self.commander = None
 
-        The method uses a try-catch block to handle the Timeout error exception. It sends the command which
-        returns a reply that is logged and then check for accepted or rejected status according to SAL
-        specifications. If the command is accepted then the command begins executing. The device is polled for
-        its status until the device is idle. If the command finishes successfully then it is logged and the
+    def move_absolute(self, value):
+        """This method moves the linear stage absolutely by the number of
+        steps away from the starting position.
```

Comment:
steps = millimeters

The word "steps" is a bit dangerous as it makes people think about stepper motors (which is what is used to actually drive the stage). The API certainly commands "Steps" - but the CSC doesn't.

Reply:
Yes, but that describes the hardware api.